core/include/thread.h

```diff
@@ -11,6 +11,97 @@
  * @ingroup     core
  * @brief       Support for multi-threading
  *
+ * Priorities
+ * ==========
```

Comment:
Good point. ^^ I've added ` * For more information about priorities and scheduling, see @ref core_sched.
`